### ISSUE TITLE
The XHTML transformation fails with NullPointerException when the entire map is chunked

### ISSUE THREAD

**Issue by balasaalin** [dita-ot/dita-ot#2730]
- I believe this to be a bug, not a question about using DITA-OT.
- I read the [CONTRIBUTING][] file.

## Actual Behavior
I have a DITA map with the @chunk='to-content' attribute set on the "map" element. The map contains a "topichead" which is also chunked. When I try to transform the map to XHTML, the transformation fails with the following NullPointerException:

```
BUILD FAILED
D:\projects\trunk\eXml\frameworks\dita\DITA-OT2.x\build.xml:45: The following error occurred while executing this line:
D:\projects\trunk\eXml\frameworks\dita\DITA-OT2.x\plugins\org.dita.base\build_preprocess.xml:314: java.lang.NullPointerException
	at org.dita.dost.writer.ChunkTopicParser.processChunk(ChunkTopicParser.java:238)
	at org.dita.dost.writer.ChunkTopicParser.write(ChunkTopicParser.java:60)
	at org.dita.dost.reader.ChunkMapReader.processCombineChunk(ChunkMapReader.java:536)
	at org.dita.dost.reader.ChunkMapReader.processTopicref(ChunkMapReader.java:341)
	at org.dita.dost.reader.ChunkMapReader.chunkMap(ChunkMapReader.java:227)
	at org.dita.dost.reader.ChunkMapReader.process(ChunkMapReader.java:136)
	at org.dita.dost.writer.AbstractDomFilter.read(AbstractDomFilter.java:55)
	at org.dita.dost.reader.ChunkMapReader.read(ChunkMapReader.java:118)
	at org.dita.dost.module.ChunkModule.execute(ChunkModule.java:80)
	at org.dita.dost.pipeline.PipelineFacade.execute(PipelineFacade.java:70)
	at org.dita.dost.invoker.ExtensibleAntInvoker.execute(ExtensibleAntInvoker.java:222)
	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:293)
	at sun.reflect.GeneratedMethodAccessor6.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
	at org.apache.tools.ant.Task.perform(Task.java:348)
	at org.apache.tools.ant.Target.execute(Target.java:435)
	at org.apache.tools.ant.Target.performTasks(Target.java:456)
	at org.apache.tools.ant.Project.executeSortedTargets(Project.java:1405)
	at org.apache.tools.ant.helper.SingleCheckExecutor.executeTargets(SingleCheckExecutor.java:38)
	at org.apache.tools.ant.Project.executeTargets(Project.java:1260)
	at org.apache.tools.ant.taskdefs.Ant.execute(Ant.java:441)
	at org.apache.tools.ant.taskdefs.CallTarget.execute(CallTarget.java:105)
	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:293)
	at sun.reflect.GeneratedMethodAccessor6.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
	at org.apache.tools.ant.Task.perform(Task.java:348)
	at org.apache.tools.ant.Target.execute(Target.java:435)
	at org.apache.tools.ant.Target.performTasks(Target.java:456)
	at org.apache.tools.ant.Project.executeSortedTargets(Project.java:1405)
	at org.apache.tools.ant.Project.executeTarget(Project.java:1376)
	at org.apache.tools.ant.helper.DefaultExecutor.executeTargets(DefaultExecutor.java:41)
	at org.apache.tools.ant.Project.executeTargets(Project.java:1260)
	at org.apache.tools.ant.Main.runBuild(Main.java:854)
	at org.apache.tools.ant.Main.startAnt(Main.java:236)
	at org.apache.tools.ant.launch.Launcher.run(Launcher.java:287)
	at org.apache.tools.ant.launch.Launcher.main(Launcher.java:113)
```

## Steps to Reproduce

1. Open the attached DITA map in oXygen.
2. Run the "DITA Map XHTML" transformation scenario.
[npe.zip](https://github.com/dita-ot/dita-ot/files/1090738/npe.zip)

## Environment

* DITA-OT version: 
  2.4.4
* Operating system and version _(Linux, macOS, Windows)_: 
  Windows 10 Pro
* How did you run DITA-OT?
   oXygen 19
* Transformation type _(HTML5, PDF, custom, etc.)_:  
  XHTML

[CONTRIBUTING]: https://github.com/dita-ot/dita-ot/blob/develop/.github/CONTRIBUTING.md

**Comment by robander** [comment]
NPE occurs at this line, because `output` is null: https://github.com/dita-ot/dita-ot/blob/develop/src/main/java/org/dita/dost/writer/ChunkTopicParser.java#L239

Adding a `if (output != null)` clause around that lets the build complete, but I get no output from the attached samples -- probably related to the general issue that `topichead` isn't properly creating stub topics when chunked.

**Comment by robander** [comment]
Fixed with #3216.
